airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/destination/StreamSyncSummary.kt

```diff
@@ -3,17 +3,16 @@
  */
 package io.airbyte.cdk.integrations.destination
 
-import java.util.*
+import io.airbyte.protocol.models.v0.AirbyteStreamStatusTraceMessage.AirbyteStreamStatus
+import java.util.Optional
 
 /**
  * @param recordsWritten The number of records written to the stream, or empty if the caller does
  * not track this information. (this is primarily for backwards-compatibility with the legacy
  * destinations framework; new implementations should always provide this information). If this
  * value is empty, consumers should assume that the sync wrote nonzero records for this stream.
  */
-data class StreamSyncSummary(val recordsWritten: Optional<Long>) {
-
-    companion object {
-        @JvmField val DEFAULT: StreamSyncSummary = StreamSyncSummary(Optional.empty())
-    }
-}
+data class StreamSyncSummary(
+    val recordsWritten: Optional<Long>,
```

Comment:
Do we need the optional?

Reply:
oh good point, this was some dumb backwards-compat thing for non-async dv2 destinations but I think there's none of those. I'll remove it